src/content/notero-item.ts

```diff
@@ -53,6 +53,10 @@ export default class NoteroItem {
     return this.zoteroItem.getField('date') || null;
   }
 
+  public getDateAdded(): string | null {
+    return this.zoteroItem.getField('dateAdded') || null;
```

Comment:
We can use Zotero's built-in `sqlToDate` function to convert this to a JavaScript `Date` object with the correct time zone.

```suggestion
  public getDateAdded(): Date | null {
    return Zotero.Date.sqlToDate(this.zoteroItem.dateAdded, true) || null;
```